python/source/org/libreoffice/ide/eclipse/python/PythonWizardPage.java

```diff
@@ -1,125 +0,0 @@
- * Foundation, Inc., 59 Temple Place, Suite 330, Boston,
- * MA 02111-1307 USA
- *
- * The Initial Developer of the Original Code is: Sun Microsystems, Inc..
- *
- * Copyright: 2002 by Sun Microsystems, Inc.
- *
- * All Rights Reserved.
- *
- * Contributor(s): Cedric Bosdonnat
- *
- *
- ************************************************************************/
-package org.libreoffice.ide.eclipse.python;
-
-import org.eclipse.swt.SWT;
-import org.eclipse.swt.layout.GridData;
-import org.eclipse.swt.layout.GridLayout;
-import org.eclipse.swt.widgets.Composite;
-import org.libreoffice.ide.eclipse.core.gui.rows.BooleanRow;
-import org.libreoffice.ide.eclipse.core.gui.rows.FieldEvent;
-import org.libreoffice.ide.eclipse.core.gui.rows.IFieldChangedListener;
-import org.libreoffice.ide.eclipse.core.model.UnoFactoryData;
-import org.libreoffice.ide.eclipse.core.model.language.LanguageWizardPage;
-
-/**
- * Java configuration page of the new UNO project wizard.
- */
-public class PythonWizardPage extends LanguageWizardPage {
-
-    public static final String JAVA_VERSION = "java_version"; //$NON-NLS-1$
-    public static final String JAVA_TESTS = "java_tests"; //$NON-NLS-1$
-
-    private BooleanRow mJavaTestsRow;
-
-    private boolean mUseTests;
-
-    /**
-     * Constructor.
-     */
-    public PythonWizardPage() {
-        super();
-        setImageDescriptor(OOoJavaPlugin.getDefault().getImageRegistry().getDescriptor(OOoJavaPlugin.WIZBAN));
-        setTitle(Messages.getString("PythonWizardPage.PageTitle")); //$NON-NLS-1$
-        setDescription(Messages.getString("PythonWizardPage.PageDescription")); //$NON-NLS-1$
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public void setProjectInfos(UnoFactoryData pData) {
-        // default value
-        mUseTests = true;
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public UnoFactoryData fillData(UnoFactoryData pData) {
-
-        if (pData != null) {
-            pData.setProperty(JAVA_VERSION, "java5"); // default java version
-            pData.setProperty(JAVA_TESTS, mUseTests);
-        }
-
-        return pData;
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public void createControl(Composite pParent) {
-
-        Composite body = new Composite(pParent, SWT.NONE);
-        body.setLayout(new GridLayout(1, false));
-        body.setLayoutData(new GridData(GridData.FILL_BOTH));
-
-        // Create the test row
-        mJavaTestsRow = new BooleanRow(body, JAVA_TESTS,
-            Messages.getString("PythonWizardPage.IncludeTestClasses")); //$NON-NLS-1$
-        mJavaTestsRow.setValue(true);
-        mJavaTestsRow.setFieldChangedListener(new IFieldChangedListener() {
-
-            @Override
-            public void fieldChanged(FieldEvent pEvent) {
-                mUseTests = mJavaTestsRow.getBooleanValue();
-            }
-        });
-
-        setControl(body);
-    }
-}
```

Comment:
Since there are no test classes for the Python Project which can be used during the time of creation of New Project, therefore we do not require the (second) page in the New Project where we include the test classes, it can be seen if you go for New Extension Project Wizard using Java as the language